### ISSUE TITLE
Unicode lat long in yelp businesses aborting event crawling.

### ISSUE THREAD

**Issue by jhugman** [mozilla-mobile/prox-server#73]
```
Dec 02 15:20:26 ip-172-31-12-95 python[18051]: ERROR:prox:Error researching venue
Dec 02 15:20:26 ip-172-31-12-95 python[18051]: Traceback (most recent call last):
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:   File "app/request_handler.py", line 130, in researchEvent
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:     eventObj = getEventfulEventObj(eventfulObj)
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:   File "app/request_handler.py", line 229, in getEventfulEventObj
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:     yelpId = _guessYelpId(event['venue_name'], locLat, locLng)
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:   File "app/request_handler.py", line 197, in _guessYelpId
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:     biz = min(r.businesses, key=lambda b:
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:   File "app/request_handler.py", line 199, in <lambda>
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:     (b.location.coordinate.latitude, b.location.coordinate.longitude))
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:   File "app/geo.py", line 44, in distance
Dec 02 15:20:26 ip-172-31-12-95 python[18051]:     latDelta = degreesToRadians(location2[0] - location1[0]);
Dec 02 15:20:26 ip-172-31-12-95 python[18051]: TypeError: unsupported operand type(s) for -: 'float' and 'unicode'
```

**Comment by jhugman** [comment]
These seems to be happening with eventful events.

**Comment by liuche** [comment]
This should be fixed by #71, where we explicitly set the type for lat/lng.
